canonical/ubuntu-pro-client
# Publish docs from docs-devel

## Why is this needed?

This publishes docs that were staged in `docs-devel` until release. Now that v34 is out we can publish them.




## Test Steps

These were already reviewed once before merging into `docs-devel`. Check the PR build to make sure the new docs still look correct.




---
